project/src/utils.ts

```diff
@@ -1,3 +1,26 @@
-import { MAX_STARS_RATING, FIVE_STARS_WIDTH } from './consts';
+import { MAX_STARS_RATING, FIVE_STARS_WIDTH, SelectOptions } from './consts';
+import { Offer } from './types/offer';
 
 export const getRatingStarsProcent = (rating: number) => (FIVE_STARS_WIDTH / MAX_STARS_RATING) * rating;
+
+export const compareOffer = (high:number, low:number) => high - low;
+
+export const sortList = (sortType: string, list: Offer[]) => {
+  const { DEFAULT, HIGH_TO_LOW, LOW_TO_HIGT, TOP_RATED } = SelectOptions;
+
+  if (sortType !== DEFAULT) {
```

Comment:
```suggestion

  if (sortType !== DEFAULT) {
    return
  }
```

а потом уже не вложенные на первом уровне условия. 

а еще можно сделать объект, в котором полями будут название сортировки, а в значении функция.